charts/cp-zookeeper/values.yaml

```diff
@@ -48,6 +48,11 @@ autoPurgePurgeInterval: 24
 ## Zookeeper JVM Heap Option
 heapOptions: "-Xms512M -Xmx512M"
 
+## Additional env variables
+customEnv: {}
```

Comment:
@cesarokuti hi
other charts use configOverrides convention. 
kindly follow it here. 

https://github.com/confluentinc/cp-helm-charts/blob/bc75a82288e3b9fe9970b8886e5914e5bb0e68e0/charts/cp-ksql-server/templates/deployment.yaml#L93

Reply:
The other charts has 2 options like:
https://github.com/confluentinc/cp-helm-charts/blob/master/charts/cp-schema-registry/templates/deployment.yaml#L91

Reply:
Maybe are two diferent options like Schema Registry or Kafka configurations:

https://github.com/confluentinc/cp-helm-charts/blob/bc75a82288e3b9fe9970b8886e5914e5bb0e68e0/charts/cp-kafka/templates/statefulset.yaml#L128

Reply:
These commit add two options like cp-kafka chart.